crates/bevy_log/src/lib.rs

```diff
@@ -166,7 +172,17 @@ impl Plugin for LogPlugin {
 
         let default_filter = { format!("{},{}", self.level, self.filter) };
         let filter_layer = EnvFilter::try_from_default_env()
-            .or_else(|_| EnvFilter::try_new(&default_filter))
+            .or_else(|from_env_error| {
+                if let Some(err) = from_env_error
+                    .source()
+                    .unwrap()
+                    .downcast_ref::<ParseError>()
+                {
+                    eprintln!("LogPlugin failed to parse filter from env: {}", err);
```

Comment:
Can we use an `error!` macro here instead?

Reply:
Nope, using `error!` doesn't produce any logs here. I think this is because the logger is not ready at this point.

Reply:
Great, I thought that might be the case. Can you add a comment explaining this so the next poor fool doesn't waste 5 minutes trying this?